lib/pushprocessor.js

```diff
@@ -298,3 +306,11 @@ function PushProcessor(client) {
 
 /** The PushProcessor class. */
 module.exports = PushProcessor;
+
+/**
+ * See above
+ * @param {array} actionlist The actions list
+ *
+ * @return {object} See above
+ */
+module.exports.actionListToActionsObject = actionListToActionsObject;
```

Comment:
instead of this dance, why not define it directly with `PushProcessor.actionListToActionsObject = function(...) {...}` (and update the references elsewhere in the file accordingly)?